src/Chapter2.hs

```diff
@@ -336,7 +348,9 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList start end xs
+        | start < 0 || end < 0 = []
```

Comment:
Would something like below be better?
```
subList :: Int -> Int -> [a] -> [a]
subList start end xs
        | start < 0   = []
        | end <= 0  = []  
        | otherwise = drop start . take (end + 1) $ xs
```

Reply:
Your new implementation is slightly better. But what I mean is that you can additionally check if `end > start` and return an empty list in this case as well

Reply:
I am not sure if I follow but maybe it is a typo; shouldn't `end` be higher than `start` since end would be of higher index? Anyway, I fixed it a bit:
```suggestion
        | start < 0     = []
        | end <= 0    = []  
        | end < start = []
        | otherwise   = drop start . take (end + 1) $ xs
```

Reply:
Yeah, that's the correct fix 🙂 You got the idea 👍🏻

Reply:
Thanks!